### ISSUE TITLE
High altitude exposure

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10144]
**Preferred term label:**
High altitude exposure

**Synonyms**
Hiking, mountain climbing, travel to mountains, air travel

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C1384955

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Eustachian tube dysfunction
Mountain sickness, acute

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000033
- label: High altitude exposure
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: A history of being at high altitude in situations such hiking, mountain climbing, travel to mountains, or air travel.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10144
- synonyms:
